sdk/storage/Azure.Storage.DataMovement.Blobs/CHANGELOG.md

```diff
@@ -26,6 +26,8 @@
     - `BlobStorageResourceOptions.CacheControl`
     - `BlobStorageResourceOptions.Metadata`
 - Changed `BlobStorageResourceContainerOptions.BlobType` from `Azure.Storage.Blobs.Models.BlobType` to `DataTransferProperty<Azure.Storage.Blobs.Models.BlobType>`
+- Changed `FromContainer(string containerUri, BlobStorageResourceContainerOptions options = default)` to `FromContainer(Uri containerUri, BlobStorageResourceContainerOptions options = default)`
+- Changed `FromBlob(string blobUri, BlobStorageResourceOptions options = default)` to `FromBlob(Uri blobUri, BlobStorageResourceOptions options = default)`
```

Comment:
You accidentally added this to the wrong part of the changelog. This needs to go under `
## 12.0.0-beta.6 (Unreleased)`

Reply:
Oops, fixed